tests/ui/parser/mismatched-delimiter-corner-case-issue-127868.rs

```diff
@@ -0,0 +1,7 @@
+// ignore-tidy-trailing-newlines
```

Comment:
Is this necessary? It seems like having a newline at the end of the file wouldn't hurt this test.